### ISSUE TITLE
Dane: TGE (Towarowa Giełda Energii)

### ISSUE THREAD

**Issue by home409ca** [jacek2511/ha_rce#10]
Z tego co wiem to Operatorzy do wyznaczania stawek używają "TGE - Rynek Dnia Następnego - Fixing I", a nie tak jak w dodatku jest z PSE.

Czy można dodać wybór źródła danych?

Cena na dzień bieżący dostępna jest tutaj: https://tge.pl/energia-elektryczna-rdn lub tutaj https://energy.instrat.pl/ceny/energia-rdn-godzinowe/

Ps. Dlaczego zdecydowałeś się na dane z PSE a nie TGE?

**Comment by miqm** [comment]
PSE udostępnia fajne api i to ceny wyznaczone przez PSE a nie TGE są cenami po której sprzedaje się prąd w net-billingu.

**Comment by senderbell** [comment]
Z tego co się zorientowałem to cena z TGE będzie uwzględniana przy obliczaniu ceny stosowanej przy taryfie dynamicznej dla osób bez fotowoltaiki. Przynajmniej tak jest napisane na stronie PGE. Jak widać prosumenci są rozliczani na nieco innych zasadach. Byłoby super gdyby można było jednak wybrać źródło aktualnej ceny energii.

**Comment by nerodoxus** [comment]
tak, zgadza się - dla PGE tu są podane wartości rozliczeń dla taryfy dynamicznej https://www.gkpge.pl/dla-domu/oferta/dynamiczna-energia-z-pge#pane-table. Są zbieżne z TGE Fixing I - https://www.tge.pl/electricity-dam

**Comment by nerodoxus** [comment]
Tutaj jest API z cenami TGE - https://energy-instrat-api.azurewebsites.net/api/prices/energy_price_rdn_hourly?date_from=20-10-2024T00:00:00Z&date_to=20-10-2024T23:59:59Z
